apps/condo/domains/organization/components/EmployeeProfileForm/index.tsx

```diff
@@ -98,7 +122,7 @@ export const EmployeeProfileForm = () => {
                                         name={'role'}
                                         label={RoleLabel}
                                     >
-                                        <EmployeeRoleSelect employee={employee}/>
+                                        <EmployeeRoleSelect employeeRoles={employeeRoles} error={employeeRolesError ? ErrorMessage : undefined} />
```

Comment:
employeeRolesError contains error or null. Why do you need a ternary operator here?